config/vufind/FeedbackForms.yaml

```diff
@@ -246,7 +246,7 @@ forms:
 
       - name: boolean
         type: radio
-        label: Need Help?
+        label: Need Help
```

Comment:
This needs to be changed to help_header, doesn't it?

Reply:
As I renamed the tokens this one would be more like `feedback_forms_label_need_help` although I'm still hesitant as this schema looks like it is more or less introducing domains in translation tokens (`feedback_forms_...`).

Reply:
Since this particular configuration is just an example, I think it's hard to justify creating a whole new translation string that will, in likelihood, never get used by more than one or two instances. It might be arguable that reusing help_header as an example makes sense in that context, or that we don't need to use real working translation strings at all (though I don't much like that approach). I'm curious if @EreMaijala has an opinion about this.

Reply:
I don't have a strong opinion, but I'd prefer example configuration that works. Otherwise it can be confusing to new users. The problem with using an existing translation is that anyone changing that may not realize it's used elsewhere as well, but that's a risk I'd be willing to take (and it's the same with many other translation strings as well).

Reply:
`help_header` right now is not used at all so it would be a specific translation string for this occurrence anyway. As you suggest `feedback_title` in the other comment I would argue contextualizing the token more to the specific use with `feedback_help_label` would be more fitting than `help_header`. In the feedback form the tokens have a very specific context, adjusting the translations for these contexts would very often mean introducing new tokens or strings. So providing specific strings would be more comfortable and minimize the risk of unwillingly changing translation in one place and causing confusion in another place.
In the end I think this comes to the question if at some point we want to tokenize all translation strings (as far as it goes).

Reply:
If `help_header` is not currently being used anywhere, I'd definitely support renaming that to `feedback_help_label` for clarity.